modules/admin_manual/pages/maintenance/upgrade.adoc

```diff
@@ -13,12 +13,16 @@ Before beginning an upgrade, please keep the following points in mind:
 * Review xref:release_notes.adoc[the release notes] for important information
 about the needed migration steps during that upgrade to help ensure a
 smooth upgrade process.
+* Check ownCloud’s Mandatory Requirements
+ownCloud's xref:installation/manual_installation#requirements[mandatory requirements] (such as PHP versions and extensions) can change from one version to the next.
+Ensure that you review them and update your server(s), if required, before upgrading ownCloud.
 * Skipping major releases is not supported. However _you can_ migrate
-from 9.0.9 straight to 10.0.
+from latest 8.2.x and latest 9.0.x straight to latest ownCloud 10.x.x
```

Comment:
These are the exact versions where an upgrade straight to 10 is possible. 
8.2.11
9.0.9
9.1.X

Reply:
What about:
`However _you can_ migrate straight forward to the latest ownCloud 10 release from either 8.2.11 or 9.0.9 or 9.1.x.`

Reply:
This is intentional, it might occur that we need to backport due to security issues. Then we would have another minor release for the old versions. The latest minor for 9.0 is is 9.0.11 by the way. I would prefer to have the "latest" as that matches all cornercases and we won't have to touch that in case of a backport.